test/integration/framework/master_utils.go

```diff
@@ -141,16 +141,18 @@ func startMasterOrDie(masterConfig *master.Config) (*master.Master, *httptest.Se
 		}
 
 		masterConfig = &master.Config{
-			DatabaseStorage:    etcdStorage,
-			ExpDatabaseStorage: expEtcdStorage,
-			KubeletClient:      client.FakeKubeletClient{},
-			EnableLogsSupport:  false,
-			EnableProfiling:    true,
-			EnableUISupport:    false,
-			APIPrefix:          "/api",
-			ExpAPIPrefix:       "/experimental",
-			Authorizer:         apiserver.NewAlwaysAllowAuthorizer(),
-			AdmissionControl:   admit.NewAlwaysAdmit(),
+			DatabaseStorage:      etcdStorage,
+			ExpDatabaseStorage:   expEtcdStorage,
+			KubeletClient:        client.FakeKubeletClient{},
+			EnableExp:            true,
```

Comment:
The way it is right now, KUBE_API_VERSIONS just registers the API objects and conversion funcs. Master does not expose the experimental API in apiserver until EnableExp variable here is set to true: https://github.com/kubernetes/kubernetes/blob/master/pkg/master/master.go#L573